src/index.native.tsx

```diff
@@ -91,8 +105,9 @@ let NativeFullWindowOverlay: React.ComponentType<View>;
 
 const ScreensNativeModules = {
   get NativeScreen() {
-    NativeScreenValue =
-      NativeScreenValue || requireNativeComponent('RNSScreen');
+    const nativeComponent = () =>
```

Comment:
Is there a reason this code is wrapped in a function?

Reply:
So that it is not executed twice

Reply:
So shouldn't we do it in current implementation too?